LFGBulletinBoard/Options.lua

```diff
@@ -251,6 +251,7 @@ function GBB.OptionsInit ()
 	GBB.Options.AddButton(GBB.L["BtnUnselectAll"],function()
 		DoSelectFilter(false, WotlkChkBox_FilterDungeon, GBB.WOTLKDUNGEONSTART, GBB.WOTLKMAXDUNGEON)
 	end)
+	GBB.Options.AddDrop(GBB.DB,"InviteRole", "DPS", {"DPS", "Tank", "Healer"})
```

Comment:
Can you add a space/newline before this? Right now the position is a bit awkward

Reply:
Done. Had to change an offset in LibGPIOptions.